.github/workflows/format-code.yml

```diff
@@ -7,6 +7,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: ministryofjustice/github-actions/code-formatter@v6
+      - uses: ministryofjustice/github-actions/code-formatter@v18.2.2
```

Comment:
That's a big jump!

Reply:
It's their latest version, I think we stuck to v6 because [the README didn't update to the latest version](https://github.com/ministryofjustice/bichard7-next-core/pull/1008).